.github/actions/demo/macos/action.yml

```diff
@@ -0,0 +1,28 @@
+name: macOS
+description: Build for macOS
+
+inputs:
+  GITHUB_TOKEN:
+    required: true
+    description: On macos runners we often hit API limits. Using this token avoid this.
+
+runs:
+  using: "composite"
+  steps:
+    - uses: extractions/setup-just@v1
+      env:
+        GITHUB_TOKEN: ${{ inputs.GITHUB_TOKEN }}
+    - name: Install toolchain
+      shell: bash
+      run: just default-toolchain
+    - uses: Swatinem/rust-cache@v1
+    # TODO: Build in release instad of debug doesn't work
+    # TODO: Sign app
```

Comment:
I think we should not sign our demos. We should find a workaround for this. We do not want to use non-personal certificates.
Can you create an issue?

Reply:
What could be the workaround for this? I can remove the comment but I don't think we can prevent macOS from warning about security risks without signing.

Reply:
```suggestion
```